### PR TITLE
#167190404 Share articles on social media

### PR DESCRIPTION
**Pull request by chuxmykel**
#### What does this PR do?
- This PR enables the user to share articles via email or on social media.

#### Description of Task to be completed?
- integrate with social media platforms to share articles

#### How should this be manually tested?
- visit the review app link created for this PR and select an article to read. Scroll down to the bottom of the page and hover over the options icon (3 dots) and select the platform you want to share in.

#### Any background context you want to provide?
- [e2e test](https://drive.google.com/file/d/1PQXkzmg__wd0x1Gp5RBi3qUeCSZU8Bl_/view?usp=sharing)
#### What are the relevant pivotal tracker stories?
- [#167190404](https://www.pivotaltracker.com/story/show/167190404)
#### Screenshots (if appropriate)
-

### CODE REVIEW

**Review by Hadeneekeh — APPROVED** [review]
LGTM!!!

File: src/views/Articles/ReadArticle/ReadArticle.js [hunk 1]
```diff
@@ -70,6 +80,10 @@ export class ReadArticle extends Component {
       });
   }
 
+  handleClick = name => {
+    document.querySelector(`[aria-label="${name}"]`).click();
```
Comment (Lundii) [inline src/views/Articles/ReadArticle/ReadArticle.js:154]: Nice work Chuks. Can you please throw more light on this part of this code, I looked at the ReadArticle file but could not find a selector "aria-label". Thank you

Comment (chuxmykel) [reply src/views/Articles/ReadArticle/ReadArticle.js:154]: Thanks for very much for this question @lundii. You see, the share button didn't occupy the whole span in which I wrapped it in. I had to use this line to transfer the click to the share button when any part of the span is clicked.

Comment (Lundii) [reply src/views/Articles/ReadArticle/ReadArticle.js:154]: > Thanks for very much for this question @Lundii. You see, the share button didn't occupy the whole span in which I wrapped it in. I had to use this line to transfer the click to the share button when any part of the span is clicked.

Alright. Nice one

**Review by Lundii — APPROVED** [review]

**Review by cvjude — APPROVED** [review]
LGTM

**Review by Lundii — APPROVED** [review]

**Review by obayomi96 — APPROVED** [review]
Good job @chuxmykel

**Review by chiboycalix — APPROVED** [review]
LGTM

**Pull request merged**